k2/torch/csrc/fsa_class_test.cu

```diff
@@ -22,49 +22,48 @@
 
```

Comment:
Could you also add tests for other functions, e.g.,
- SetScores
- SetAttr/GetAttr/DeleteAttr
- Scores, Labels
- Tests for fillers